docs/concepts-messaging.md

```diff
@@ -515,16 +515,64 @@ Exclusive is the default subscription type.
 
 #### Failover
 
-In the *Failover* type, multiple consumers can attach to the same subscription. A master consumer is picked for a non-partitioned topic or each partition of a partitioned topic and receives messages. When the master consumer disconnects, all (non-acknowledged and subsequent) messages are delivered to the next consumer in line.
+In the *Failover* type, multiple consumers can attach to the same subscription. 
 
-* For partitioned topics, the broker will sort consumers by priority level and lexicographical order of consumer name. The broker will try to evenly assign partitions to consumers with the highest priority level.
-* For non-partitioned topics, the broker will pick consumers in the order they subscribe to the non-partitioned topics.
+A master consumer is picked for a non-partitioned topic or each partition of a partitioned topic and receives messages. 
 
-For example, a partitioned topic has 3 partitions, and 15 consumers. Each partition will have 1 active consumer and 4 stand-by consumers.
+When the master consumer disconnects, all (non-acknowledged and subsequent) messages are delivered to the next consumer in line.
 
-In the diagram below, **Consumer A** is the master consumer while **Consumer B** would be the next consumer in line to receive messages if **Consumer A** is disconnected.
+##### Failover | Partitioned topics
```

Comment:
For reviewers: 

To improve legibility and readability, I add `|` in 2 headings because 4-level headings and 5-leave headings look the same size on the webpage (this is a Docusarus limitation).